packages/block-library/src/navigation/index.php

```diff
@@ -898,4 +898,75 @@ function block_core_navigation_typographic_presets_backcompatibility( $parsed_bl
 	return $parsed_block;
 }
 
+/**
+ * Build an array with CSS classes and inline styles defining the colors
+ * which will be applied to the navigation markup in the front-end.
+ *
+ * @param  array $context    Navigation block context.
+ * @param  array $attributes Block attributes.
+ * @return array Colors CSS classes and inline styles.
+ */
+function block_core_navigation_submenu_build_css_colors( $context, $attributes ) {
```

Comment:
Should we change the name to not reference `submenu` here?

Reply:
```suggestion
function block_core_navigation_build_css_colors( $context, $attributes ) {
```